Spatial_Engine/Query/PointLayout.cs

```diff
@@ -164,7 +164,7 @@ public static List<Point> PointLayout(this LinearLayout layout2D, IEnumerable<IC
         /***************************************************/
 
 
-        [Description("Returns Points along straight lines throughout the region. If the line is segemented by openings or the region curve the points will be distributed to the different segments based on their length. \n" +
+        [Description("Returns Points along one or more straight lines through the region curves. If the line is discontinuous because of openings or concave host region, Points will be distributed to the different segments based on their length.\n" +
```

Comment:
```suggestion
        [Description("Returns Points along one or more straight lines through the region curves. If the line is discontinuous because of openings, a concave host region, or multiple host regions, Points will be distributed to the different segments based on their length.\n" +
```